ggml-cuda.cu

```diff
@@ -255,6 +294,23 @@ static void dequantize_row_q8_0_cuda(const void * vx, float * y, int k, cudaStre
     dequantize_block_q8_0<<<nb, 1, 0, stream>>>(vx, y);
 }
 
+static void dequantize_mul_mat_q4_0_cuda(const void * vx, const float * y, float * dst, const int ncols, const int nrows, cudaStream_t stream) {
+    // static int block_size = -1;
```

Comment:
I would suggest to remove commented code.

Reply:
Already done for the version that I intend to get merged: https://github.com/ggerganov/llama.cpp/pull/1412